tests/reading-tests.md

```diff
@@ -0,0 +1,397 @@
+
+Reading the line from left-to-right, it seems like `expect(result)` is also a function call. The function's name is `expect` and we're passing in `result`.
+
+However, the line doesn't stop there. Leila sees `.toEqual('Fizz')`. Recalling other JavaScript syntax, Leila suspects that this is a function call, where the function is named `toEqual` and the argument is `'Fizz'`.
+
+It looks like this function call operates from an object. That object must be `expect(result)`.
+
+### Reading it Out Loud
+
+Leila can now take a step back and read this line overall. They notice a sentence that the code forms: "Expect the value of `result` to equal `'Fizz'`."
+
+The test will pass if `result`'s value is `'Fizz'`, and fail if it isn't.
+
+## Multiple Tests in One Describe Block
+
+Leila just can't catch a break! After reading through the first test, Nadia is prompting Leila to continue reading.
+
+However, Leila can use this opportunity to possibly validate or disprove some of their theories.
+
+<!-- prettier-ignore-start -->
+```js
+describe('fizzBuzz', () => {
+  test('returns Fizz on multiples of 3', () => {
+    // Arrange
+    const num = 9;
+
+    // Act
+    const result = fizzBuzz(num);
+
+    // Assert
+    expect(result).toEqual('Fizz');
+  });
+
+  test('returns Buzz on multiples of 5', () => {
+    expect(fizzBuzz(25)).toEqual('Buzz');
+  });
+});
+```
+<!-- prettier-ignore-end -->
+
+The first thing Leila notices is that the next test is a second `test` block, located in the same `describe` block.
+
+There aren't commas between the tests, because these tests are not in an object with comma-separated key-value pairs, nor are they in a comma-separated array.
+
+The `test` block continues to be, ultimately, a function call: `test(..., ...);`
+
+### First Argument of `test`
+
+Leila sees that the first argument is another string literal, `'returns Buzz on multiples of 5'`. Now, Leila can draw a comparison to the other test, which passed in `'returns Fizz on multiples of 3'`.
+
+Leila lets themself feel a little more confident in hypothesizing. They hypothesize that the first argument is a string that describes the intent of the test.
+
+### Second Argument of `test`
+
+Now, what's left for Leila to consider is the anonymous function.
+
+<!-- prettier-ignore-start -->
+```js
+() => {
+    expect(fizzBuzz(25)).toEqual('Buzz');
+}
+```
+<!-- prettier-ignore-end -->
+
+Drawing also from the first unit test, Leila concludes:
+
+1. This assert step begins with calling `expect(...)` and passing in a value.
+   - In the first test, the value was `result`, which was ultimately `fizzBuzz(9)`
+   - In this second test, the value is `fizzBuzz(25)`
+1. The line continues, chaining on `.toEqual(...)`
+   - In the first test, the argument was `'Fizz'`
+   - In this second test, the argument is `'Buzz'`
+
+To summarize, this second test expects the value of `fizzBuzz(25)` to equal `'Buzz'`.
+
+## Example on Replit.com
```

Comment:
There seems to be a big problem with the Replit. The view of the replit that a non-owner sees is that summary view with the green button (I made a suggestion in the learning another language PR to include an image of that screen with some direction to Fork the replit to get to the same view as the the remaining images).

That should be fine foe most cases, but either the replit here is missing its test (strange since you got screenshots), OR forking does not bring the unit tests along... 😱😱😱😱😱

Is there a permission setting or something that can be toggled on the adacore repls so that they end up in the owner view, but without permanent change permissions? Is there a problem with the unit tests on this replit? If not, there's going to be a problem with distributing an environment that has tests using replit.